includes/class-wc-stripe-helper.php

```diff
@@ -1983,4 +1983,24 @@ public static function validate_intent_for_order( $order, $intent, ?string $sele
 
 		throw new Exception( __( "We're not able to process this request. Please try again later.", 'woocommerce-gateway-stripe' ) );
 	}
+
+	/**
+	 * Determines if the store is connected to Stripe.
+	 *
+	 * @param string $mode Optional. The mode to check. 'live' or 'test' - if not provided, the currently enabled mode will be checked.
+	 * @return bool True if connected, false otherwise.
+	 */
+	public static function is_connected( $mode = null ) {
```

Comment:
Also, from a naming perspective, is it worth calling this function something more like `have_stripe_keys()`? The "connected" wording implies that we have a connection to Stripe, but in this case we only know that we have saved keys, not whether they actually work.